app_dart/lib/src/request_handlers/flaky_handler_utils.dart

```diff
@@ -446,21 +448,25 @@ BuilderType getTypeForBuilder(String? builderName, YamlMap ci) {
   return hasFrameworkTag && hasHostOnlyTag ? BuilderType.frameworkHostOnly : BuilderType.unknown;
 }
 
-List<dynamic>? _getTags(String? builderName, YamlMap ci) {
-  final YamlList targets = ci[kCiYamlTargetsKey] as YamlList;
-  final YamlMap? target = targets.firstWhere(
-    (dynamic element) => element[kCiYamlTargetNameKey] == builderName,
-    orElse: () => null,
-  ) as YamlMap?;
+List<String>? _getTags(String? targetName, CiYaml ciYaml, {bool unfilteredTargets = false}) {
+  List<Target> allTargets;
+  if (!unfilteredTargets) {
+    allTargets = ciYaml.presubmitTargets;
+    allTargets.addAll(ciYaml.postsubmitTargets);
+  } else {
+    allTargets = ciYaml.targets;
```

Comment:
Do we want it as all targets? Such as ones that do not include LUCI or Cocoon? Or should custom targets also have TESTOWNERS?

(Looking today, there's no custom targets, but we may add them back in the future)

Reply:
Oh.. what are custom targets? I am assume these do not run with either of the supported schedulers based on your statement. In that case where would these run?
I can add changes to support that if it is valuable at this point in time.

Reply:
https://github.com/flutter/cocoon/blob/main/CI_YAML.md#external-tests

Right now there's none beyond LUCI and Cocoon, but it's a reasonable infra request for us to add them back. Theoretically, we could add Google testing here, or any other CI system the team may want to use.

Reply:
Ah okay so we would have a need to filter by scheduler at least.